google-cloud-contrib/google-cloud-logging-logback/README.md

```diff
@@ -0,0 +1,134 @@
+Logback Appender for Google Cloud Logging
+=========================================
+
+Logback appender `com.google.cloud.logging.logback.LoggingAppender` for
+[Google Cloud Logging](https://cloud.google.com/logging/).
+
+This library allows you to use [Logback](https://logback.qos.ch/) configuration and the [Slf4j](https://www.slf4j.org/) interface to log via Google Cloud logging
+
+-  [Homepage](https://googlecloudplatform.github.io/google-cloud-java/)
+-  [API Documentation](https://googlecloudplatform.github.io/google-cloud-java/apidocs/index.html?com/google/cloud/logging/package-summary.html)
+
+> Note: This client is a work-in-progress, and may occasionally
+> make backwards-incompatible changes.
+
+Quickstart
+----------
+If you are using Maven, add this to your pom.xml file
+```xml
+<dependency>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-logging-logback</artifactId>
+  <version>0.17.1-alpha</version>
+</dependency>
+```
+If you are using Gradle, add this to your dependencies
+```Groovy
+compile 'com.google.cloud:google-cloud-logging-logback:0.17.1-alpha'
+```
+If you are using SBT, add this to your dependencies
+```Scala
+libraryDependencies += "com.google.cloud" % "google-cloud-logging-logback" % "0.17.1-alpha"
+```
+
+Usage 
+-----
+
+Add the appender to your [Logback configuration](https://logback.qos.ch/manual/configuration.html) `logback.xml`.
+See [Logback filters](https://logback.qos.ch/manual/filters.html#thresholdFilter) on filtering log output and
+ [encoders](https://logback.qos.ch/manual/encoders.html) on formatting.
+
+
+```xml
+<configuration>
+  <appender name="CLOUD" class="com.google.cloud.logging.logback.LoggingAppender">
+    <!-- Optional : filter logs at or above a level -->
+    <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+          <level>INFO</level>
+     </filter>
+    <log>application.log</log> <!-- Optional : default java.log -->
+    <enhancers>com.example.enhancers.TestLoggingEnhancer</enhancers> <!-- Optional -->
+    <flushLevel>WARN</flushLevel> <!-- Optional : default ERROR -->
+  </appender>
+
+  <root level="info">
+    <appender-ref ref="CLOUD" />
+  </root>
+</configuration>
+```
+
+In your code :
+
+```java
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+ 
+public class TestLogger {
+    private final Logger logger = LoggerFactory.getLogger(TestLogger.class);
+ 
+    public void log(String name) {
+        logger.info("This is a test");
+    }
+
+    public static void main(String[] args) {
+        TestLogger testLogger = new TestLogger();
+        testLogger.log("test");
+    }
+}
+```
+
+
+Authentication
+--------------
+
+See the [Authentication](https://github.com/GoogleCloudPlatform/google-cloud-java#authentication)
+section in the base directory's README.
+
+
+Limitations
+-----------
+
+This library is usable, but not yet complete.
+ * Auto-reload of properties is not fully tested
```

Comment:
Maybe instead provide a link to an issues search with a label filter (create a label for logging-logback)